### ISSUE TITLE
Runtime checks for essential packages from `base-devel` group

### ISSUE THREAD

**Issue by actionless** [actionless/pikaur#138]

**Comment by djsavvy** [comment]
I apologize if I'm misunderstanding this issue, but couldn't this just be solved by adding base-devel to the 'depends' array of pikaur's PKGBUILD?

**Comment by actionless** [comment]
no, 1) group can't be a dependency ; 2) by AUR packaging rules packages from base-devel shouoldn't be mentioned as deps because they assumed to be installed ni order to use AUR

**Comment by djsavvy** [comment]
Thanks for explaining. What's the point of runtime checks for packages from `base-devel` then if they need to be installed already to use the AUR?

**Comment by actionless** [comment]
it can run without installation (for example for bootstrapping a freshly installed system or container)

**Comment by AladW** [comment]
Do you really want to appeal to a user base that hasn't even done the basic step of installing Arch from the installation guide? (if they had, they would know that base-devel is required for AUR usage)

**Comment by AladW** [comment]
The requirement on base-devel (in particular fakeroot being installed) is an AUR convention, which is Arch-only. `pacman` is supposed to work on other distributions (or even macOS or Windows), where such convention may not be present.

**Comment by actionless** [comment]
btw i was thinking of adapting it to work msys2 but i am wondering if there are any AUR packages for msys2? or they could have their own MSYS2-UR...

**Comment by AladW** [comment]
I have been wondering on these things too, it's sort of a shame to put in all these efforts and it only works for one distribution (well I guess in my case due to modularity you can reuse some parts that are unrelated to `aur.archlinux.org`). 

Perhaps a more natural candidate would be [pacbsd](https://pacbsd.org/) and a "PUR" (meow).

**Comment by actionless** [comment]
i've mentioned msys2 because i am actually using it myself, however so far it seems what the only change need to be done is make AUR URL configurable (which can be also helpful when using local AUR server for testing/debugging)
